src/components/transactions/Repay/CollateralRepayModalContent.tsx

```diff
@@ -88,7 +130,11 @@ export function CollateralRepayModalContent({
     .multipliedBy(poolReserve.priceInUSD)
     .multipliedBy(100 + Number(maxSlippage))
     .dividedBy(100)
-    .dividedBy(collateralReserveData.priceInUSD);
+    .dividedBy(
+      collateralReserveData && collateralReserveData.priceInUSD
+        ? collateralReserveData.priceInUSD
+        : poolReserve.priceInUSD
+    );
```

Comment:
We shouldn't factor in slippage for the amount of collateral required when repaying with a tokens. Maybe here it just makes sense to handle it with an if statement, something like:
```
let collateralAmountRequiredToCoverDebt = safeAmountToRepayAll ...
if (! repaying with atoken) {
  const collateralAmountRequiredToCoverDebt = safeAmountToRepayAll
       .multipliedBy(poolReserve.priceInUSD)
       .multipliedBy(100 + Number(maxSlippage))
       .dividedBy(100)
}
```